frontend/src/tabs/zoom.pug

```diff
@@ -31,15 +38,19 @@
       h3(v-else) {{ day.date }}
       ul
         li(v-for="slice in day.commitResults").commit-message.active
-          a(v-bind:href="getSliceLink(slice)" target="_blank") {{ slice.messageTitle }}
-          span &nbsp; ({{ slice.insertions }} lines)
-          a(
-            v-if="slice.messageBody !== ''",
-            v-on:click="updateExpandedCommitMessagesCount",
-            onclick="toggleNext(this)",
-            title="Click to show/hide the commit message body"
-          )
-            i.commit-message--button.fas.fa-ellipsis-h
+          .title
+            a(v-bind:href="getSliceLink(slice)" target="_blank") {{ slice.messageTitle }}
+            span &nbsp; ({{ slice.insertions }} lines) &nbsp;
+            .tag(v-if="slice.tag", v-bind:class="[`${slice.hash}`]")
```

Comment:
Why do you need to bind the hash to this tag?

Reply:
Because I have a method allows users to scroll to the specific commit (based on the html class name) when the tag is clicked.  I used the hash as the classname because the hash is unique, don't need to worry about duplicates.

https://github.com/reposense/RepoSense/blob/4a63761c17967ef5e14722130b0da3fccf048410/frontend/src/static/js/v_zoom.js#L45-L50

Reply:
OK